tests/test_onnxml_one_hot_encoder_converter.py

```diff
@@ -51,6 +51,33 @@ def test_one_hot_encoder_onnx_int(self, rtol=1e-06, atol=1e-06):
         # Check that predicted values match
         np.testing.assert_allclose(onnx_ml_pred, onnx_pred, rtol=rtol, atol=atol)
 
+    # Test OneHotEncoder with 2 inputs
+    @unittest.skipIf(
+        not (onnx_ml_tools_installed() and onnx_runtime_installed()), reason="ONNXML test requires ONNX, ORT and ONNXMLTOOLS"
```

Comment:
What prompted this addition? I do not expect to see onnx-ml in this PR; am I missing something?

Reply:
I think I added this because a test using onnx-ml for one-hot encoding was missing. We were only directly converting to onnx using HB.